.github/workflows/release.yml

```diff
@@ -164,11 +166,39 @@ jobs:
       - name: Show collected files
         run: ls -al dist
 
+      - name: Decide tag
+        id: tag_name
+        run: |
+          if [ '${{ github.event_name }}' = 'schedule' ]; then
+            # Check if any nightly release already exists for this SHA
+            SHORT_SHA="${{ github.sha }}"
```

Comment:
This could also be stored in `env`:

```yaml
env:
  COMMIT_SHA: ${{ github.sha }}
```

And then:

```suggestion
            SHORT_SHA="$COMMIT_SHA"
```

I think this way it's more readable